ts/src/test/static/request/digifinex.json

```diff
@@ -714,6 +714,32 @@
                   "USDT"
                 ]
             }
+        ],
+        "transfer": [
```

Comment:
@yzh-pelle can you add a test here, transferring from swap to spot?

Reply:
done